overlord/ifacestate/udevmonitor/udevmon.go

```diff
@@ -88,8 +91,15 @@ func (m *Monitor) Connect() error {
 		return fmt.Errorf("cannot start udev monitor: %s", err)
 	}
 
-	// TODO: consider passing a device filter to reduce noise from irrelevant devices.
-	m.monitorStop = m.netlinkConn.Monitor(m.netlinkEvents, m.netlinkErrors, nil)
+	var filter netlink.Matcher
+	// TODO: extend with other criteria based on the hotplug interfaces
```

Comment:
This is slightly unrelated to this PR it seem(?). Its ok but I couldn't help noticing. I guess because this goes directly to netlinkConn.Monitor() we cannot add a test that the subsystem matching does really work(?).

Reply:
Indeed, this should be part of another PR, removed.